server/util/body-parser.js

```diff
@@ -110,6 +111,46 @@ export function convertWpList(node) {
   }
 }
 
+export function findWpImageGallery(node) {
```

Comment:
I've used `find` here to potentially start thinking about splitting it into finders and converters.